config.ru

```diff
@@ -3,4 +3,11 @@
 # This file is used by Rack-based servers to start the application.
 
 require ::File.expand_path('../config/environment', __FILE__)
+
+#require 'rack'
+require ::File.expand_path('../lib/monitoring/middleware/conjur_exporter.rb', __FILE__)
+
+#use Rack::Deflater
+use Monitoring::Middleware::ConjurExporter
```

Comment:
Looks like we already have a dedicated place for applying middleware, see https://github.com/cyberark/conjur/blob/master/config/initializers/rack_middleware.rb#L24-L28. I think we would want to have similar code as we do for `::Rack::DefaultContentType` so that the exporter middleware runs before anything else.

Reply:
Good point - I imagine this is especially true if using middleware to collect metrics on incoming requests, in case something further down the middleware chain were to modify the request. Do you think the same precedence is needed for the exporter middleware?